install/0000_30_machine-api-operator_12_baremetalhost.crd.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: baremetalhosts.metal3.io
+spec:
+  group: metal3.io
+  names:
+    kind: BareMetalHost
+    listKind: BareMetalHostList
+    plural: baremetalhosts
+    singular: baremetalhost
+    shortNames:
+     - bmh
+     - bmhost
+  scope: Namespaced
+  version: v1alpha1
```

Comment:
are we good shipping with alpha commitment?

Reply:
For this high touch beta, yes. We'll raise the version level in an upcoming release.

Reply:
alpha is fine.